clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalCatalog.java

```diff
@@ -239,12 +235,11 @@ public boolean purgeTable(NameIdentifier ident) throws UnsupportedOperationExcep
               ErrorHandlers.tableErrorHandler());
       resp.validate();
       return resp.dropped();
-
-    } catch (UnsupportedOperationException e) {
-      throw e;
     } catch (Exception e) {
-      LOG.warn("Failed to purge table {}", ident, e);
-      return false;
```

Comment:
it's not correct to return false

Reply:
if so, could you please add a todo comment on int